2025-03-09 19:07:02,015 INFO    StreamThr :23580 [internal.py:wandb_internal():86] W&B internal server running at pid: 23580, started at: 2025-03-09 19:07:02.014352
2025-03-09 19:07:02,018 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: status
2025-03-09 19:07:02,062 INFO    WriterThread:23580 [datastore.py:open_for_write():85] open: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_190701-dyf1bhv3\run-dyf1bhv3.wandb
2025-03-09 19:07:02,067 DEBUG   SenderThread:23580 [sender.py:send():382] send: header
2025-03-09 19:07:02,163 DEBUG   SenderThread:23580 [sender.py:send():382] send: run
2025-03-09 19:07:02,902 INFO    SenderThread:23580 [dir_watcher.py:__init__():211] watching files in: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_190701-dyf1bhv3\files
2025-03-09 19:07:02,902 INFO    SenderThread:23580 [sender.py:_start_run_threads():1136] run started: dyf1bhv3 with start time 1741527422.014351
2025-03-09 19:07:02,933 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: check_version
2025-03-09 19:07:02,934 DEBUG   SenderThread:23580 [sender.py:send_request():409] send_request: check_version
2025-03-09 19:07:03,287 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: run_start
2025-03-09 19:07:04,449 DEBUG   HandlerThread:23580 [system_info.py:__init__():32] System info init
2025-03-09 19:07:04,449 DEBUG   HandlerThread:23580 [system_info.py:__init__():47] System info init done
2025-03-09 19:07:04,450 INFO    HandlerThread:23580 [system_monitor.py:start():194] Starting system monitor
2025-03-09 19:07:04,450 INFO    SystemMonitor:23580 [system_monitor.py:_start():158] Starting system asset monitoring threads
2025-03-09 19:07:04,451 INFO    HandlerThread:23580 [system_monitor.py:probe():214] Collecting system info
2025-03-09 19:07:04,464 INFO    SystemMonitor:23580 [interfaces.py:start():190] Started cpu monitoring
2025-03-09 19:07:04,466 INFO    SystemMonitor:23580 [interfaces.py:start():190] Started disk monitoring
2025-03-09 19:07:04,468 INFO    SystemMonitor:23580 [interfaces.py:start():190] Started gpu monitoring
2025-03-09 19:07:04,481 INFO    SystemMonitor:23580 [interfaces.py:start():190] Started memory monitoring
2025-03-09 19:07:04,516 INFO    SystemMonitor:23580 [interfaces.py:start():190] Started network monitoring
2025-03-09 19:07:04,547 DEBUG   HandlerThread:23580 [system_info.py:probe():196] Probing system
2025-03-09 19:07:04,551 DEBUG   HandlerThread:23580 [system_info.py:_probe_git():181] Probing git
2025-03-09 19:07:04,726 DEBUG   HandlerThread:23580 [system_info.py:_probe_git():189] Probing git done
2025-03-09 19:07:04,726 DEBUG   HandlerThread:23580 [system_info.py:probe():244] Probing system done
2025-03-09 19:07:04,726 DEBUG   HandlerThread:23580 [system_monitor.py:probe():223] {'os': 'Windows-10-10.0.26100-SP0', 'python': '3.9.5', 'heartbeatAt': '2025-03-09T13:37:04.548723', 'startedAt': '2025-03-09T13:37:01.986350', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': '<python with no main file>', 'codePathLocal': None, 'git': {'remote': 'git@github.com-work:saimanikumar-da24m016/da6401_assignment1.git', 'commit': '3ddbfd47c7127abba95926229ee5c61c7e5e5eb9'}, 'email': 'saimanikumar67@gmail.com', 'root': 'C:/Users/D. Sai Mani Kumar/Downloads/DL/da6401_assignment1', 'host': 'LAPTOP-VRI27UL2', 'username': 'D. Sai Mani Kumar', 'executable': 'c:\\Users\\D. Sai Mani Kumar\\AppData\\Local\\Programs\\Python\\Python39\\python.exe', 'cpu_count': 4, 'cpu_count_logical': 8, 'cpu_freq': {'current': 2400.0, 'min': 0.0, 'max': 2400.0}, 'cpu_freq_per_core': [{'current': 2400.0, 'min': 0.0, 'max': 2400.0}], 'disk': {'/': {'total': 375.82519149780273, 'used': 262.47074127197266}}, 'gpu': 'NVIDIA GeForce GTX 1650 Ti', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce GTX 1650 Ti', 'memory_total': 4294967296}], 'memory': {'total': 7.840869903564453}}
2025-03-09 19:07:04,726 INFO    HandlerThread:23580 [system_monitor.py:probe():224] Finished collecting system info
2025-03-09 19:07:04,726 INFO    HandlerThread:23580 [system_monitor.py:probe():227] Publishing system info
2025-03-09 19:07:04,726 DEBUG   HandlerThread:23580 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2025-03-09 19:07:04,728 DEBUG   HandlerThread:23580 [system_info.py:_save_pip():68] Saving pip packages done
2025-03-09 19:07:04,730 INFO    HandlerThread:23580 [system_monitor.py:probe():229] Finished publishing system info
2025-03-09 19:07:04,763 DEBUG   SenderThread:23580 [sender.py:send():382] send: files
2025-03-09 19:07:04,763 INFO    SenderThread:23580 [sender.py:_save_file():1392] saving file wandb-metadata.json with policy now
2025-03-09 19:07:04,811 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 19:07:04,811 DEBUG   SenderThread:23580 [sender.py:send_request():409] send_request: stop_status
2025-03-09 19:07:04,817 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 19:07:04,914 INFO    Thread-16 :23580 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_190701-dyf1bhv3\files\wandb-metadata.json
2025-03-09 19:07:04,915 INFO    Thread-16 :23580 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_190701-dyf1bhv3\files\requirements.txt
2025-03-09 19:07:05,220 DEBUG   SenderThread:23580 [sender.py:send():382] send: telemetry
2025-03-09 19:07:05,221 DEBUG   SenderThread:23580 [sender.py:send():382] send: telemetry
2025-03-09 19:07:05,221 DEBUG   SenderThread:23580 [sender.py:send():382] send: run
2025-03-09 19:07:05,537 INFO    SenderThread:23580 [sender.py:send_run():1010] updated run: dyf1bhv3
2025-03-09 19:07:05,919 INFO    Thread-16 :23580 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_190701-dyf1bhv3\files\config.yaml
2025-03-09 19:07:05,920 INFO    Thread-16 :23580 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_190701-dyf1bhv3\files\output.log
2025-03-09 19:07:06,076 INFO    wandb-upload_0:23580 [upload_job.py:push():131] Uploaded file C:\Users\D9110~1.SAI\AppData\Local\Temp\tmpfnfipfxzwandb\vc51wicv-wandb-metadata.json
2025-03-09 19:07:06,631 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:06,632 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:07,563 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 19:07:07,923 INFO    Thread-16 :23580 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_190701-dyf1bhv3\files\output.log
2025-03-09 19:07:09,078 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:09,078 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:11,509 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:11,510 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:12,616 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 19:07:13,939 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:13,940 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:16,378 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:16,379 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:17,674 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 19:07:18,891 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:18,893 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:19,828 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 19:07:19,828 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 19:07:19,830 DEBUG   SenderThread:23580 [sender.py:send_request():409] send_request: stop_status
2025-03-09 19:07:21,237 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: partial_history
2025-03-09 19:07:21,240 DEBUG   SenderThread:23580 [sender.py:send():382] send: history
2025-03-09 19:07:21,240 DEBUG   SenderThread:23580 [sender.py:send_request():409] send_request: summary_record
2025-03-09 19:07:21,244 INFO    SenderThread:23580 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-09 19:07:21,326 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:21,326 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:22,103 INFO    Thread-16 :23580 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_190701-dyf1bhv3\files\output.log
2025-03-09 19:07:22,108 INFO    Thread-16 :23580 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_190701-dyf1bhv3\files\wandb-summary.json
2025-03-09 19:07:23,273 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 19:07:23,770 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:23,771 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:26,251 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:26,252 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:28,370 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 19:07:28,708 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:28,709 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:31,144 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:31,145 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:33,513 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 19:07:33,692 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:33,693 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:34,526 INFO    Thread-16 :23580 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_190701-dyf1bhv3\files\config.yaml
2025-03-09 19:07:34,835 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 19:07:34,852 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 19:07:34,854 DEBUG   SenderThread:23580 [sender.py:send_request():409] send_request: stop_status
2025-03-09 19:07:36,168 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:36,169 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:38,634 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:38,634 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:39,275 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 19:07:43,639 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:43,640 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:44,298 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 19:07:45,104 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: partial_history
2025-03-09 19:07:45,106 DEBUG   SenderThread:23580 [sender.py:send():382] send: history
2025-03-09 19:07:45,106 DEBUG   SenderThread:23580 [sender.py:send_request():409] send_request: summary_record
2025-03-09 19:07:45,110 INFO    SenderThread:23580 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-09 19:07:45,880 INFO    Thread-16 :23580 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_190701-dyf1bhv3\files\wandb-summary.json
2025-03-09 19:07:45,880 INFO    Thread-16 :23580 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_190701-dyf1bhv3\files\output.log
2025-03-09 19:07:46,066 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:46,067 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:48,504 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:48,505 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:49,842 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 19:07:49,842 DEBUG   SenderThread:23580 [sender.py:send_request():409] send_request: stop_status
2025-03-09 19:07:49,846 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 19:07:50,149 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 19:07:50,935 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:50,935 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:53,389 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:53,390 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:55,201 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 19:07:55,822 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:55,823 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:58,252 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:07:58,253 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:08:00,250 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 19:08:00,346 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: partial_history
2025-03-09 19:08:00,348 DEBUG   SenderThread:23580 [sender.py:send():382] send: history
2025-03-09 19:08:00,348 DEBUG   SenderThread:23580 [sender.py:send_request():409] send_request: summary_record
2025-03-09 19:08:00,351 INFO    SenderThread:23580 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-09 19:08:00,689 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:08:00,689 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:08:01,224 INFO    Thread-16 :23580 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_190701-dyf1bhv3\files\wandb-summary.json
2025-03-09 19:08:02,251 INFO    Thread-16 :23580 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_190701-dyf1bhv3\files\output.log
2025-03-09 19:08:03,120 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:08:03,121 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:08:04,539 DEBUG   SystemMonitor:23580 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2025-03-09 19:08:04,540 DEBUG   SenderThread:23580 [sender.py:send():382] send: stats
2025-03-09 19:08:04,854 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 19:08:04,870 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 19:08:04,871 DEBUG   SenderThread:23580 [sender.py:send_request():409] send_request: stop_status
2025-03-09 19:08:05,591 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:08:05,591 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:08:06,249 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 19:08:08,027 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:08:08,027 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:08:10,475 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:08:10,476 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:08:11,287 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 19:08:12,928 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:08:12,929 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:08:15,381 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:08:15,383 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:08:16,328 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 19:08:19,332 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: partial_history
2025-03-09 19:08:19,336 DEBUG   SenderThread:23580 [sender.py:send():382] send: history
2025-03-09 19:08:19,336 DEBUG   SenderThread:23580 [sender.py:send_request():409] send_request: summary_record
2025-03-09 19:08:19,338 INFO    SenderThread:23580 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-09 19:08:19,660 INFO    Thread-16 :23580 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_190701-dyf1bhv3\files\wandb-summary.json
2025-03-09 19:08:19,848 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-09 19:08:19,880 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: stop_status
2025-03-09 19:08:19,882 DEBUG   SenderThread:23580 [sender.py:send_request():409] send_request: stop_status
2025-03-09 19:08:20,427 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:08:20,427 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:08:20,707 INFO    Thread-16 :23580 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250309_190701-dyf1bhv3\files\output.log
2025-03-09 19:08:22,300 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 19:08:22,872 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:08:22,872 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:08:25,319 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:08:25,319 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:08:27,361 DEBUG   HandlerThread:23580 [handler.py:handle_request():146] handle_request: status_report
2025-03-09 19:08:27,779 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-09 19:08:27,780 ERROR   gpu       :23580 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
